src/componentes/unmounted/lista-nominal/print/aux/StatusTotalConsultasValidas.jsx

```
@@ -0,0 +1,53 @@
+    ) {
+      return 3;
+    }
+  }, [values]);
+
+  return (
+    <div style={{
+      ...STYLE[colorCode],
+      width: "40%",
+      fontSize: orientacao === "retrato" && "9px"
+    }}>
+      {(colorCode == 0) && <span>-</span>}
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)